src/OpenConext/EngineBlockApiClientBundle/Value/ConsentListFactory.php

```diff
@@ -76,10 +87,17 @@ private static function createConsent($data)
             )
         );
 
+        if ($data['consent_type'] === ConsentType::TYPE_EXPLICIT) {
+            $consentType = ConsentType::explicit();
+        } else if ($data['consent_type'] === ConsentType::TYPE_IMPLICIT) {
+            $consentType = ConsentType::implicit();
+        }
```

Comment:
I don't like that this if-else statement seems to be missing an else statement. Granted, it's covered by the `Assert::choice()` call some 30(!) lines above, but I'd rather put an explicit InvalidArgumentException in an else statement. Thoughts?